### PR TITLE
Create abuh_solo.js

### PR DESCRIPTION
**Pull request by Abuh05**

### CODE REVIEW

File: scripts/abuh_solo.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  console.log(
+    "Hello Word, this is " +
+      FullName +
+      "with HNGi7 ID" +
+      ID +
+      "and email " +
+      Email +
+      "using " +
+      Language +
+      "for stage 2 track "
```
Comment (GB-12) [inline scripts/abuh_solo.js:15]: Change from track to task

**Comment by simply-alliv** [comment]
working on this

File: scripts/abuh_solo.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+function HNGi7Profile() {
```
Comment (Abuh05) [inline scripts/abuh_solo.js:1]: Okay!

**Comment by dro-1** [comment]
You can only add 1 script to the repo

**Comment by Rubylena** [comment]
please redo this immediately. ask for help here https://hngi7.slack.com/archives/C01482Z0095/p1591369072416800

**Pull request closed without merging**